src/propset/scheduleInboxPropSet.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * CDAV Library
+ *
+ * This library is part of the Nextcloud project
+ *
+ * @author Georg Ehrke
+ * @copyright 2019 Georg Ehrke <oc.list@georgehrke.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import * as NS from '../utility/namespaceUtility.js';
```

Comment:
You may just import `IETF_CALDAV`

Reply:
Valid point, but we are using `import * as NS from '../utility/namespaceUtility.js';` everywhere in this lib right now. I think we should refactor it altogether at some point.

There is some bigger maintenance work needed in this lib anyway:
- improved imports
- use standard Nextcloud eslint style
- ...